CityOfDetroit/bloom
# Fix/email jurisdiction

## Issue

- Closes #721
- [x] This change is a dependency for another issue (#701)

## Description

Email text gets strings from the `translations` database, which are keyed on jurisdiction. If there aren't translations for the given jurisdiction, sending an email fails. This PR adds an initial seed migration to populate the `translations` database for Detroit. The seed values are obtained from the [existing seed migration](https://github.com/CityOfDetroit/bloom/blob/main/backend/core/src/migration/1620660845209-seed-translation-entries.ts).

This PR also updates the `EMAIL_FROM_ADDRESS` to the currently set sender address on our dev Sendgrid account.

## Type of change


## How Can This Be Tested/Reviewed?

```
$ psql bloom
bloom=# select translations.translations from translations inner join jurisdictions on translations.jurisdiction_id=jurisdictions.id and jurisdictions.name='Detroit';
```

Alternatively, set your `.env` with the correct Sendgrid `EMAIL_FROM_ADDRESS` and API key, and create an account on your local server. You should see an email being sent.

## Checklist:
